src/main/java/org/folio/rest/impl/CirculationLogsImpl.java

```diff
@@ -0,0 +1,91 @@
+package org.folio.rest.impl;
+
+import io.vertx.core.AsyncResult;
+import io.vertx.core.Context;
+import io.vertx.core.Handler;
+import org.folio.cql2pgjson.CQL2PgJSON;
+import org.folio.cql2pgjson.exception.FieldException;
+import org.folio.rest.RestVerticle;
+import org.folio.rest.annotations.Validate;
+import org.folio.rest.jaxrs.model.LogRecord;
+import org.folio.rest.jaxrs.model.LogRecordCollection;
+import org.folio.rest.jaxrs.resource.AuditDataCirculation;
+import org.folio.rest.persist.PostgresClient;
+import org.folio.rest.persist.cql.CQLWrapper;
+import org.folio.rest.tools.utils.TenantTool;
+import org.folio.rest.tools.utils.ValidationHelper;
+
+import javax.ws.rs.core.Response;
+import java.util.Collection;
+import java.util.List;
+import java.util.Map;
+import java.util.concurrent.CompletableFuture;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+import static io.vertx.core.Future.succeededFuture;
+
+public class CirculationLogsImpl implements AuditDataCirculation {
+  protected static final String DB_TAB_FEES_FINES = "fees_fines";
+  protected static final String DB_TAB_ITEM_BLOCKS = "item_blocks";
+  protected static final String DB_TAB_LOANS = "loans";
+  protected static final String DB_TAB_MANUAL_BLOCKS = "manual_blocks";
+  protected static final String DB_TAB_NOTICES = "notices";
+  protected static final String DB_TAB_PATRON_BLOCKS = "patron_blocks";
+  protected static final String DB_TAB_REQUESTS = "requests";
+
+  @Override
+  @Validate
+  public void getAuditDataCirculationLogs(String query, String lang, Map<String, String> okapiHeaders,
+    Handler<AsyncResult<Response>> asyncResultHandler, Context vertxContext) {
+    List<CompletableFuture<List<LogRecord>>> futures = Stream.of(DB_TAB_FEES_FINES, DB_TAB_ITEM_BLOCKS,
```

Comment:
Maybe it is more useful to save db table names in the set or list? I see that they are used only once here.